realm/realm-library/src/androidTest/java/io/realm/RealmObjectTests.java

```diff
@@ -1738,7 +1738,7 @@ public void addChangeListener_returnedObjectOfCopyToRealmOrUpdate() {
 
         AllTypesPrimaryKey allTypesPrimaryKey = new AllTypesPrimaryKey();
         allTypesPrimaryKey.setColumnLong(1);
-        allTypesPrimaryKey.setColumnFloat(42f);
+        allTypesPrimaryKey.setColumnFloat(0f);
```

Comment:
it is changed on line 1756. the modification is because of the listener won't be triggered if it is added after transaction commit.